### DISCUSSION TITLE
Connect to cockroach cluster

### DISCUSSION THREAD

**Discussion by sergeymavrin** [zitadel/zitadel#7544] [Q&A] [answered]
Cannot find information how connect zitadel to cluster of cockroachdb. Everywhere in documentation - there is only one host connect to. Also tried to grep in code, but no luck.

**Comment by fforootd** [comment]
Check out this link here https://zitadel.com/docs/self-hosting/manage/database it has a tab for cockroachDB

**Reply by sergeymavrin** [reply]
There is no answer for my question there. I have 3 nodes of cockroachdb and want to have ability parse all 3 nodes to zitadel in config for disaster switch in case of unavaliability of any.

**Reply by sergeymavrin** [reply]
Maybe zitadel haven't such ability and I have to setup loadbalancer between zitadel and cockroachdb cluster?

**Reply by fforootd** [reply, marked as answer]
Ah now I understand your question. You want to instruct zitadel to connect to multiple nodes instead of a load balancer.

You need to [operate an LB in front on CockroachDB](https://www.cockroachlabs.com/docs/stable/recommended-production-settings#load-balancing).

**Reply by sergeymavrin** [reply]
ok, thanks
